config/src/main/java/com/alibaba/nacos/config/server/controller/ConfigController.java

```diff
@@ -87,31 +87,31 @@
 @RestController
 @RequestMapping(Constants.CONFIG_CONTROLLER_PATH)
 public class ConfigController {
-    
+
```

Comment:
Please do not change indent.
Please use nacos code style to reformat code.

see src/style/codeStyle.md

Reply:
ok